home/tables/columns.py

```diff
@@ -45,7 +45,7 @@ def grade_to_element(self, grade):
 
     def render(self, value: dict):
         review = value.pop("review")
-        is_staff = value.pop("is_staff")
+        is_planetterp_admin = value.pop("is_planetterp_admin")
```

Comment:
What would be involved in having this permission have the user's `is_staff=True` but `is_superuser=False`? This would allow us to continue to use the is_staff check everywhere except in places where we only want superusers, in which case we'd use `is_superuser`. This would also make it easier to search for users based on if they're staff or not.

Reply:
as you said, the issue here is the admin panel. is_staff is also a very stiff permission and would not allow us to split things into multiple permissions in the future if we wanted.